assets/src/components/detours/diversionPanel.tsx

```diff
@@ -66,6 +66,15 @@ export const DiversionPanel = ({
                   {d.instruction}
                 </ListGroup.Item>
               ))}
+              {detourFinished && (
+                <ListGroup.Item
+                  key="return-to-route"
```

Comment:
suggestion: would this still work if we dropped the `key` prop? AFAIK I don't think we need it since we're not deriving this from a list.
```suggestion
```